examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -34,7 +35,10 @@ class Task extends TypedEventEmitter<ITaskEvents> implements ITask {
     public constructor(
         private readonly _id: string,
         private readonly _name: SharedString,
-        private readonly _priority: ISharedCell<number>
+        private readonly _priority: ISharedCell<number>,
+        public incomingName: string = NONE_INCOMING_NAME,
```

Comment:
Should these new properties be externally editable? Or should they be public get, private set?

Reply:
yep changed to getter/setter pattern